src/core/or/circuitmux.c

```diff
@@ -561,19 +561,8 @@ circuitmux_set_policy(circuitmux_t *cmux,
  */
 
 cell_direction_t
-circuitmux_attached_circuit_direction(circuitmux_t *cmux, circuit_t *circ)
+circuitmux_attached_circuit_direction(chanid_circid_muxinfo_t *hashent)
```

Comment:
If we're changing the interface of this function we should change its documentation too.